public/app/plugins/datasource/mixed/MixedDataSource.ts

```diff
@@ -98,6 +98,13 @@ export class MixedDatasource extends DataSourceApi<DataQuery> {
     return Promise.resolve({});
   }
 
+  getQueryDisplayText(query: DataQuery) {
```

Comment:
@ifrost For the comment about wanting more information other than the type for mixed datasources, this is going to be tricky and I've changed my mind about adding in the datasource name for now. We create the "datasource type: datasource text" string for query history here, but we only have access to what is in the query right now. Putting in a call to get other datasource information requires an `await` and the interface for datasources does not allow this function to be async. We could parse this string apart in query history, do the datasource look up there, and add it in the middle of the string, but I would rather not do that without more discussion, and I would like that to happen outside this PR.